about/changelog.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Changelog
+excerpt: A summary of the latest changes to all Timescale products.
+keywords: [changelog, upgrades, updates, releases]
+---
+
+import ReleaseNotification from "versionContent/_partials/_release_notification.mdx";
+
+# Changelog
+
+All the latest features and updates to Timescale products.
+
+## 🧙 Compression Wizard
+<Label type="date">June 13, 2024</Label>
+
+The compression wizard is now available on Timescale Cloud. Select a hypertable and be guided through enabling compression through the UI!
+
+To access the compression wizard, navigate to `Explorer`, and select the hypertable you would like to compress. In the top right corner, hover where it says `Compression off`, and open the wizard. You will then be guided through the process of configuring compression for your hypertable, and can compress it directly through the UI.
+
+![Run the compression wizard in Timescale Console](https://assets.timescale.com/docs/images/compress-data-in-console.png)
+
+## 🏎️💨 High Performance AI Apps With pgvectorscale
+
+<Label type="date">June 11, 2024</Label>
+
+The [vectorscale extension][pgvectorscale] is now available on [Timescale Cloud][signup].
+
+pgvectorscale complements pgvector, the open-source vector data extension for PostgreSQL, and introduces the
+following key innovations for pgvector data:
+
+- A new index type called StreamingDiskANN, inspired by the DiskANN algorithm, based on research from Microsoft.
+- Statistical Binary Quantization: developed by Timescale researchers, This compression method improves on
+  standard Binary Quantization.
+
+On benchmark dataset of 50 million Cohere embeddings (768 dimensions each), PostgreSQL with pgvector and
+pgvectorscale achieves 28x lower p95 latency and 16x higher query throughput compared to Pinecone's storage
+optimized (s1) index for approximate nearest neighbor queries at 99% recall, all at 75% less cost when
+self-hosted on AWS EC2.
+
+See the [pgvectorscale documentation][pgvectorscale] to learn more.
+
+## 🧐 Integrate AI Into Your Database Using pgai
+
+<Label type="date">June 11, 2024</Label>
+
+The [pgai extension][pgai] is now available on [Timescale Cloud][signup].
+
+pgai brings embedding and generation AI models closer to the database. With pgai, you can now do the following directly
+from within PostgreSQL in a SQL query:
+
+* Create embeddings for your data.
+* Retrieve LLM chat completions from models like OpenAI GPT4o.
+* Reason over your data and facilitate use cases like classification, summarization, and data enrichment on your existing relational data in PostgreSQL.
+
+See the [pgai documentation][pgai] to learn more.
+
+
+## TimescaleDB 2.15.2
```

Comment:
What would make this awesome is to turn this into marketing language like the other releases. The issue list is already in the repo and there is a link to it.

Reply:
Yeah I tend to agree here. This is a curated list, the full list can be linked.

Reply:
Who can do that? I don't have the context to convert this release into marketing prose

Reply:
Would you like me to implement these comments?

Reply:
@iroussos For all the releases on this page, can you please convert them to marketing prose?

Reply:
@billy-the-fish Sorry I didn't see your comment when I wrote my last comment. Yea sure you can give it a try! 

@iroussos maybe you can chime in as well?

Reply:
Thank you Rahil, makes sense, I'll wait 👍

Reply:
Done.